components/playlists/browser/buildflags/buildflags.gni

```diff
@@ -0,0 +1,6 @@
+import("//brave/build/config.gni")
+import("//build/config/features.gni")
+
+declare_args() {
+  enable_brave_playlists = (brave_channel == "development" || brave_channel == "nightly") && (is_mac || is_linux || is_win || is_android)
```

Comment:
why do we need a buildflag and a feature flag for this? We generally only use buildflags for platform-specific code and this appears to be enabled on all platforms

Reply:
with this configuration we also will have no idea if the build fails when disabled until the code lands in beta